easybuild/easyblocks/i/intel_compilers.py

```diff
@@ -138,8 +138,9 @@ def make_module_extra(self):
         # On Debian/Ubuntu, /usr/include/x86_64-linux-gnu, or whatever dir gcc uses, needs to be included
         # in $CPATH for Intel C compiler
         multiarch_out, ec = run_cmd("gcc -print-multiarch", simple=False)
+        multiarch_out = multiarch_out.strip()
```

Comment:
Are you sure you can do this if ec != 0, i.e. is multiarch_out never None?

Reply:
Output of running a command can never be `None`, stdout and stderr are combined, at worst you have an empty string here, so this should be OK.

And as the command is run now, you'll never actually get to this line, `run_cmd` will raise an error if the exit code is non-zero.

Reply:
Yes, as long as we do `run_cmd(simple=False)` the content of`multiarch_out` will be a string. Something like `ls > /dev/null 2&>1` will return an empty string.